src/wfassoc.c

```diff
@@ -144,7 +144,6 @@ PEXT pExtBase = NULL;
 
 INT APIENTRY AssociateFileDlgProc(register HWND hDlg, UINT wMsg, WPARAM wParam, LPARAM lParam);
 BOOL AssociateDlgInit(HWND hDlg, LPTSTR lpszExt, INT iSel);
-VOID AssoicateFileDlgCommand(HWND hDlg, WPARAM wParam, LPARAM lParam, PASSOCIATEFILEDLGINFO pAssociateFileDlgInfo);

```

Comment:
This is a misspelling that isn't used anywhere, the correct spelling has a different prototype below (`BOOL`)